src/js/options.js

```diff
@@ -32,15 +32,13 @@ if (!matches || matches[1] == "Firefox") {
 
 const USER_DATA_EXPORT_KEYS = ["action_map", "snitch_map", "settings_map"];
 
-var backgroundPage = chrome.extension.getBackgroundPage();
-var require = backgroundPage.require;
-var badger = backgroundPage.badger;
-var log = backgroundPage.log;
-var constants = backgroundPage.constants;
+var constants = require("constants");
```

Comment:
`require()` is missing from the options page now because it's no longer defined anywhere options can see. You'll need to add script tags for `boostrap.js` (where `require` is defined) and probably other scripts directly to `options.html`.

Reply:
Done